lib/apartment/adapters/abstract_adapter.rb

```diff
@@ -55,7 +54,7 @@ def current_tenant
       #   Note alias_method here doesn't work with inheritence apparently ??
       #
       def current
-        Apartment.connection.current_database
+        unenvironmentify(Apartment.connection.current_database)
```

Comment:
Tenant name should be "site_152", instead of "development_site_152". Rsapi production should not be affected tho.

Reply:
Known issue https://github.com/influitive/apartment/issues/465